apps/wallet/src/ui/app/pages/home/transfer-coin/index.tsx

```diff
@@ -117,25 +119,21 @@ function TransferCoinPage() {
             { to, amount }: FormValues,
             { resetForm }: FormikHelpers<FormValues>
         ) => {
-            if (coinType === null) {
+            if (coinType === null || !gasBudgetEstimation) {
                 return;
             }
             setSendError(null);
             trackEvent('TransferCoins', {
                 props: { coinType },
             });
             try {
-                const bigIntAmount = BigInt(
-                    new BigNumber(amount)
-                        .shiftedBy(coinDecimals)
-                        .integerValue()
-                        .toString()
-                );
+                const bigIntAmount = parseAmount(amount, coinDecimals);
```

Comment:
Is there any risk that `amount` fails to parse resulting in 0 here? I can't imagine there is but just figured I'd ask.

Reply:
There are but it will fail for the validation as well so probably it wont be a problem